README.md

```diff
@@ -330,7 +331,7 @@ fastlane prod
 1. `bundle exec fastlane tag` to create a tag and bump the build number. You'll be prompted to enter those release notes you wrote. (:wq to save and exit)
 1. `bundle exec fastlane release` to build and push a release to Github
 1. `bundle exec fastlane internal` to distribute the builds to internal test groups in TestFlight and the Play Store
-1. If it's the end of the release cycle, prep an App Store release on App Store Connect using the latest build. Write custom release notes that summarize changes since the last App Store release (not since the last build). Submit for review with manual release control. Haven't figured out a good way to automate this, maybe impossible.
+1. If it's the end of the release cycle, `bundle exec fastlane prod` to prep an App Store release on App Store Connect. You'll be prompted to write custom release notes that summarize changes since the last App Store release (not since the last build). Attach the latest build to the new version and submit for review with manual release control. Haven't figured out a good way to automate this without submitting to review at the same time, maybe impossible.
```

Comment:
IMO you should be more clear that after doing `bundle exec fastlane prod`, you then need to go to App Store Connect to attach the build and finish the release. If I were trying to follow these instructions as a total newbie, I'd be wondering "where do I attach the latest build?"